tests/playbooks/tests_team_exists.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: remove the NetworkManager-team package
+      package:
+        name: "NetworkManager-team"
+        state: absent
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-team checking before team configuration"
+      debug:
+        msg: "NetworkManager-team is not installed before team configuration"
+      when: "'NetworkManager-team' not in ansible_facts.packages"
+
+    - name: "Team interface configuration"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          # Specify the team profile
+          - name: team0
+            state: up
+            type: team
+            interface_name: team0
+            # ip configuration (optional)
```

Comment:
If it is optional, why do you specify it?

Reply:
removed, originally I was thinking to provide a sample